mesonbuild/compilers/compilers.py

```diff
@@ -89,9 +89,9 @@
     [(suffix, lang) for suffix in v] for lang, v in lang_suffixes.items())))
 
 # Languages that should use LDFLAGS arguments when linking.
-LANGUAGES_USING_LDFLAGS = {'objcpp', 'cpp', 'objc', 'c', 'fortran', 'd', 'cuda'}
+LANGUAGES_USING_LDFLAGS = {'objcpp', 'cpp', 'cppm', 'objc', 'c', 'fortran', 'd', 'cuda'}
 # Languages that should use CPPFLAGS arguments when linking.
-LANGUAGES_USING_CPPFLAGS = {'c', 'cpp', 'objc', 'objcpp'}
+LANGUAGES_USING_CPPFLAGS = {'c', 'cpp', 'cppm', 'objc', 'objcpp'}
```

Comment:
These look wrong, you're adding cppm as a new *language* rather than a new file extension for an existing language?